candi/openapi/cluster_configuration.yaml

```
@@ -135,6 +135,31 @@ apiVersions:
         - "1.31"
         - "1.32"
         - "Automatic"
+      encryptionAlgorithm:
+        type: string
+        description: |
+          In Kubernetes version **1.30** and earlier, only the `RSA-2048` algorithm is supported.
+
+          Starting from version **1.31**, kubeadm use the specified asymmetric encryption algorithm
+          when generating keys and certificates for the following control-plane components:
+
+          - `apiserver`;  
+          - `apiserver-kubelet-client`;  
+          - `apiserver-etcd-client`;  
+          - `front-proxy-client`;  
+          - `etcd-server`;  
+          - `etcd-peer`;  
+          - `etcd-healthcheck-client`;  
```

```suggestion

          - `apiserver`;  
          - `apiserver-kubelet-client`;  
          - `apiserver-etcd-client`;  
          - `front-proxy-client`;  
          - `etcd-server`;  
          - `etcd-peer`;  
          - `etcd-healthcheck-client`;  
```

@maxinsky Please check the list formatting according to the styleguide.